src/events/messageReactionAdd.ts

```diff
@@ -10,7 +10,9 @@ export default async (client: Discord.Client, reaction: Discord.MessageReaction,
 
     const isNotSelf = user.id !== client.user?.id
     const isInSubmissionChannel = channel.id === process.env.PROJECT_SUBMISSIONS_CHANNEL
-    const isVoteEmoji = reaction.emoji?.id === process.env.UPVOTE_REACTION || reaction.emoji.id === process.env.DOWNVOTE_REACTION
+    const isValidEmoji = reaction.emoji?.id === process.env.UPVOTE_REACTION ||
+      reaction.emoji?.id === process.env.DOWNVOTE_REACTION ||
+      reaction.emoji?.name === process.env.PAUSE_REACTION
```

Comment:
See previous. Also, seems I had a past blunder here that slipped my mind, that optional chaining isn't necessary as `reaction.emoji` is guaranteed to exist.

```suggestion
    const isValidEmoji = emoji.id === process.env.UPVOTE_REACTION ||
      emoji.id === process.env.DOWNVOTE_REACTION ||
      emoji.name === process.env.PAUSE_REACTION
```